PCAxis.Web.Controls/Query/TableQueryCodebehind.vb

```diff
@@ -331,10 +331,10 @@ Public Class TableQueryCodebehind
         Dim appPath As String = String.Empty
         Dim context As System.Web.HttpContext = System.Web.HttpContext.Current
 
-        appPath = String.Format("{0}://{1}{2}{3}", _
-                                    context.Request.Url.Scheme, _
-                                    context.Request.Url.Host, _
-                                    IIf(context.Request.Url.Port.Equals(80), String.Empty, ":" & context.Request.Url.Port), _
+        appPath = String.Format("{0}://{1}{2}{3}",
+                                    context.Request.Url.Scheme,
+                                    context.Request.Url.Host,
+                                    IIf(context.Request.Url.Port.Equals(80) Or context.Request.Url.Port.Equals(443), String.Empty, ":" & context.Request.Url.Port),
```

Comment:
If should also be a condition on the protocol like

IF (port == 80 AND scheme == 'http') OR (port == 443 and scheme == 'https')  THEN skip adding port number

Reply:
Added a check on scheme http/https